tests/components/reolink/test_config_flow.py

```diff
@@ -414,3 +419,63 @@ async def test_dhcp_abort_flow(hass):
 
     assert result["type"] is data_entry_flow.FlowResultType.ABORT
     assert result["reason"] == "already_configured"
+
+
+async def test_http_no_repair_issue(hass):
+    """Test no repairs issue is raised when http local url is used."""
+    config_entry = MockConfigEntry(
+        domain=const.DOMAIN,
+        unique_id=format_mac(TEST_MAC),
+        data={
+            CONF_HOST: TEST_HOST,
+            CONF_USERNAME: TEST_USERNAME,
+            CONF_PASSWORD: TEST_PASSWORD,
+            CONF_PORT: TEST_PORT,
+            const.CONF_USE_HTTPS: TEST_USE_HTTPS,
+        },
+        options={
+            const.CONF_PROTOCOL: DEFAULT_PROTOCOL,
+        },
+        title=TEST_NVR_NAME,
+    )
+    config_entry.add_to_hass(hass)
+
+    with patch(
+        "homeassistant.components.reolink.host.get_url",
+        return_value="http://test_homeassistant_address",
+    ):
```

Comment:
Instead of patching out internal of the integration, configure the actual URL using `async_process_ha_core_config` and set up the integration normally to trigger this error.

Reply:
Done, that is indeed nicer.
I also had to configure a country code, otherwise also some missing country code repair issue will be raised.

Reply:
> I also had to configure a country code, otherwise also some missing country code repair issue will be raised.

Hmmm we should fix that in our codebase, as in, provide a default country in our hass fixture.

Reply:
I think that is out of scope for this PR right?

Reply:
yup

Reply:
Can you then approve this PR, or do I have to wait for someone to implement the fix for the hass fixture in the codebase.

Reply:
> Can you then approve this PR

Can you stop requesting that over an over again? :)